pkg/nimble/Makefile

```diff
@@ -61,8 +61,8 @@ nimble_transport_ram:
 nimble_controller:
 	"$(MAKE)" -C $(PDIR)/nimble/controller/src/ -f $(TDIR)/controller.mk
 
-nimble_drivers_nrf52:
-	"$(MAKE)" -C $(PDIR)/nimble/drivers/nrf52/src/ -f $(TDIR)/drivers.nrf52.mk
+nimble_drivers:
```

Comment:
I would leave this as NRF specific submodule, as it is only valid for these CPUs at the moment. So how about `nimble_drivers_nrf5x`?!